library/build.gradle

```diff
@@ -5,6 +5,7 @@ plugins {
     id 'bio.terra.pfb.java-library-conventions'
     id 'com.github.davidmc24.gradle.plugin.avro' version '1.9.1'
     id 'me.champeau.jmh' version '0.7.3'
+    id 'com.google.cloud.artifactregistry.gradle-plugin' version '2.1.5'
```

Comment:
The latest version is `2.2.5`, but that requires java 21 and so the project failed to build.  It seemed safer to keep the java version for this PR; if we want to update java we can increase this version as well